packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -126,14 +127,37 @@ router.post(
   "/update-all/commonwell",
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getUUIDFrom("query", req, "cxId").orFail();
-    const { patientIds = [] } = updateAllSchema.parse(req.body);
+    const { patientIds } = updateAllSchema.parse(req.body);
```

Comment:
This was defaulting to an empty array if we didnt pass the patientIds which we define as optional for this endpoint. This led to no patients being queries when really we want all the cx's patients to be queried